### PR TITLE
Implemented sitewide H tag design. Resolves issue 232.

### LINKED ISSUE #232: Sitewide review of H1, H2, H3, H4, H5 styles + A tag underline
For non-navigation regions of the site:
Make one CSS style for each H tag. Make available a CSS class for a change in color.

Remove underline from all A tags regardless of region.

### PR DESCRIPTION
**Pull request by jbotwins**
This work resolves #232 .

Please include a description of the work.

### Previews

Large screens

Small screens

### CODE REVIEW

**Comment by adborden** [comment]
This also fixes #234

File: _sass/module/_referendum.scss [hunk 1]
```diff
@@ -19,21 +7,22 @@
...
 
-.referendum__region {
-  h2 {
-    text-transform: uppercase;
-    color: $color-grey-2;
+.l-section__content {
+  .uppercase {
+    font-weight: $font-weight-normal;
+  }
+}
```
Comment (adborden) [inline _sass/module/_referendum.scss]: This rule should be moved to `_sass/layout/_section.scss`.

File: _sass/module/_referendum.scss [hunk 2]
```diff
@@ -19,21 +7,22 @@
...
+
+.section-header {
```
Comment (adborden) [inline _sass/module/_referendum.scss]: We should probably rename this class so it is specific to the referendum page/component

**Comment by adborden** [comment]
@anlawyer good catch on the alignment. Let's open a separate issue for that.

**Comment by adborden** [comment]
... and if you don't mind, would you update the headings to sentence case instead of title case? They are not names, and we use sentence case for most other headings on the site.

**Comment by adborden** [comment]
Hey @jbotwins let me know if you need a hand with this.

**Comment by jbotwins** [comment]
Okey dokey. Rebased, then added padding to H titles. Renamed H titles class names. Removed all caps titles.

**Pull request merged**